src/rt/dmain2.d

```diff
@@ -558,6 +558,35 @@ private void formatThrowable(Throwable t, scope void delegate(in char[] s) nothr
     }
 }
 
+private auto parseExceptionOptions()
+{
+    static struct ExceptionOptions
+    {
+        bool trap;
+
+        @nogc nothrow void initialize()
+        {
+            import rt.config : rt_configOption;
+            trap = rt_trapExceptions;
+            auto res = rt_configOption("extrap");
```

Comment:
`exceptionTrap` might be an alternative. Which one do you like better?

Reply:
I'd prefer `trapException`? I think it's better to be a bit more verbose for clarity.